pkg/users/account-create-dialog.js

```diff
@@ -116,6 +116,15 @@ function validate_username(username, accounts) {
     return null;
 }
 
+function validate_real_name(real_name) {
+    if (!real_name)
+        return _("No real name specified");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17950-20221125-124129-c16b147c-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)